lib/storage.js

```diff
@@ -145,10 +154,8 @@ Storage.prototype.getDataBatch = function (start, n, cb) {
 }
 
 Storage.prototype.getNode = function (index, cb) {
-  if (this.cache) {
-    var cached = this.cache.get(index)
-    if (cached) return cb(null, cached)
-  }
+  var cached = this.cache.tree.get(index)
```

Comment:
cache is always defined now?

Reply:
can be null no?

Reply:
Added a test for disabled caching + added checks here